R/pkg/R/context.R

```diff
@@ -435,3 +435,31 @@ setCheckpointDir <- function(directory) {
   sc <- getSparkContext()
   invisible(callJMethod(sc, "setCheckpointDir", suppressWarnings(normalizePath(directory))))
 }
+
+#' Total number of CPU cores of all executors registered in the cluster at the moment.
+#'
+#' @rdname spark.numCores
+#' @return current number of cores in the cluster.
+#' @examples
+#'\dontrun{
+#' spark.numCores()
+#'}
+#' @note spark.numCores since 2.4.0
+spark.numCores <- function() {
+  sc <- getSparkContext()
+  invisible(callJMethod(sc, "numCores"))
```

Comment:
`invisible` in other cases are because they return void in java, in these cases we are returning numbers, and should not use `invisible`